datatypes.go

```diff
@@ -447,32 +449,49 @@ func (schedule *ScheduleData) IsScheduleAllows(ts int64) bool {
 }
 
 func (event NotificationEvent) String() string {
-	return fmt.Sprintf("TriggerId: %s, Metric: %s, Values: %s, OldState: %s, State: %s, Message: '%s', Timestamp: %v", event.TriggerID, event.Metric, event.GetMetricsValues(), event.OldState, event.State, event.CreateMessage(nil), event.Timestamp)
+	return fmt.Sprintf("TriggerId: %s, Metric: %s, Values: %s, OldState: %s, State: %s, Message: '%s', Timestamp: %v", event.TriggerID, event.Metric, event.GetMetricsValues(false), event.OldState, event.State, event.CreateMessage(nil), event.Timestamp)
 }
 
 // GetMetricsValues gets event metric value and format it to human readable presentation
-func (event NotificationEvent) GetMetricsValues() string {
-	var targetNames []string //nolint
+func (event NotificationEvent) GetMetricsValues(isLess bool) string {
```

Comment:
`isLess` — maybe we should rename it somehow? Now it's too difficult to understand the purpose of the variable from it's name.

Maybe we sould introduse an enumeration instead of a boolean flag, so that it would be more readable and could be expanded in the future

Reply:
I agree, I've refactored it now